src/Image/Image.php

```diff
@@ -271,14 +270,14 @@ public function setBorderRadius(int $cornerRadius): self
     }
 
     /**
-     * @param float opacity The opacity of the image
+     * @param  ?float  $opacity The opacity of the image
      * @return Image
      *
      * @throws \ImagickException
      */
-    public function setOpacity(float $opacity): self
+    public function setOpacity(?float $opacity = null): self
```

Comment:
Probably because this PR updated the method to include an `is_null` check, same with `setRotation`, previously it was an [`empty` check](https://github.com/utopia-php/image/blob/f8f217f49aab465e84bfacb086e7c33845e7e56e/src/Image/Image.php#L281) but @christyjacob4 pushed https://github.com/neelshah2409/image/commit/3fef4d9b43df947883a060b93cef68da5a34bd52 to update it to `is_null`.